packages/language-kit/package.json

```diff
@@ -8,17 +8,21 @@
     "url": "git+https://github.com/logto-io/toolkit.git"
   },
   "license": "MIT",
-  "type": "module",
   "source": "./src/index.ts",
-  "main": "./lib/index.cjs",
-  "exports": "./lib/index.js",
+  "main": "./lib/index.js",
```

Comment:
Not sure, but should this be `cjs`?

Reply:
no i removed `"type": "module"` now it can be js

Reply:
but you are right cjs should work as well